### PR TITLE
Use open-source version of Ursula (Floaty)

### PR DESCRIPTION
**Pull request by ccremer**
* replaces occurrences of `Ursula` with `Floaty`
* Downloads a tagged public release from GitHub (instead from bootstrap bucket)

### CODE REVIEW

**Comment by ccremer** [comment]
PR Draft status: Do not merge yet, Floaty isn't yet released

File: lb.tf [hunk 1]
```diff
@@ -126,8 +126,8 @@ resource "exoscale_compute" "lb" {
     ],
     "runcmd" = [
       "while lsof -F p /var/lib/dpkg/lock 2>/dev/null; do echo \"Waiting for dpkg lock...\"; sleep 15; done",
-      "curl -Lo /tmp/ursula.deb ${trimsuffix(var.bootstrap_bucket, "/")}/ursula.deb",
-      "dpkg -i /tmp/ursula.deb",
+      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/download/v1.0.0/floaty_linux_amd64.deb",
```
Comment (ccremer) [inline lb.tf:129]: I'm not sure if we want to use a pinned version of the release, or whether we should make the effort to always download the latest stable version (may require more or alternative GitHub API calls)

**Review by simu — APPROVED** [review]
LGTM

File: lb.tf [hunk 2]
```diff
@@ -126,8 +126,8 @@ resource "exoscale_compute" "lb" {
     ],
     "runcmd" = [
       "while lsof -F p /var/lib/dpkg/lock 2>/dev/null; do echo \"Waiting for dpkg lock...\"; sleep 15; done",
-      "curl -Lo /tmp/ursula.deb ${trimsuffix(var.bootstrap_bucket, "/")}/ursula.deb",
-      "dpkg -i /tmp/ursula.deb",
+      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/download/v1.0.1/floaty_linux_amd64.deb",
```
Comment (simu) [inline lb.tf:129]: Automatically fetching the latest release would be possible with

```suggestion
      "curl -Lo /tmp/floaty.deb https://github.com/vshn/floaty/releases/latest/download/floaty_linux_amd64.deb",
```

Comment (ccremer) [reply lb.tf:129]: TIL something new :)

**Pull request merged**